miso-web/src/main/webapp/scripts/hot_sample.js

```diff
@@ -38,7 +38,12 @@ HotTarget.sample = (function() {
   };
   
   var getExternalNames = function(identityAlias){
-    return new Set(identityAlias.replace(/.*--(\s*)/, "").toLowerCase().split(","));
+    var externalNames = {};
+    identityAlias.replace(/.*--(\s*)/, "").toLowerCase().split(",").forEach(function(name){
+      externalNames[name] = null;
+    });
```

Comment:
I think you could simplify this a bit:
`return identityAlias.replace(/.*--(\s*)/, "").toLowerCase().split(",");`
and then your check below becomes
`return existingExternalNames.indexOf(externalNames) != -1;`